CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+## 1.0.0 (April 10, 2023)
+
+NOTES:
+
+* The Go SDK is now generally available with semantic versioning compatibility promises.
+* Now using CC API version 2023-04-10.
+
+BREAKING CHANGES:
+
+* The following enums have been renamed, adding a "Type" suffix:
+  * ApiCloudProvider (renamed to CloudProviderType)
+  * AwsEndpointConnectionStatus
```

Comment:
`SetAwsEndpointConnectionState` appears in this list twice. We can remove both and instead say something like:
- `AwsEndpointConnectionStatus` 
  - `SetAwsEndpointConnectionStatusType` is a new enum that includes the settable statuses from `AwsEndpointConnectionStatusType`.

Also `CockroachCloudSetAwsEndpointConnectionStateRequest` is renamed to `SetAwsEndpointConnectionStateRequest`.